### DISCUSSION TITLE
I can't finish compiling.

### DISCUSSION THREAD

**Discussion by nedlog-92** [KadeArchive/Kade-Engine#589] [Q&A]
![ss1](https://user-images.githubusercontent.com/84843528/119641810-04fb2e80-be55-11eb-8d42-50f3a5a4a724.png)
I've tried to compile it many times, but it just stays in the process. I have waited for as long as 10 hours, but it never finishes. Since I'm not getting any errors, I'm assuming there's nothing wrong with the configuration of my environment and it's working fine.

I'm running Windows 7.
I have created a template with flixel tpl -n "Test" and run lime test neko, html5 and windows.
As a result, neko and html5 were successful, and windows was in the same state as above.
Does anyone else have the same problem or know of a solution?
![zdrhy](https://user-images.githubusercontent.com/84843528/119642472-b8fcb980-be55-11eb-84b5-302a0f4f78c7.png)
![rarsuuw](https://user-images.githubusercontent.com/84843528/119642483-bc904080-be55-11eb-92f2-c7684e27d6e7.png)
![fxjd](https://user-images.githubusercontent.com/84843528/119642489-be5a0400-be55-11eb-953a-5ea31917c68b.png)

**Comment by isakube** [comment]
i thiiiiink windows 7 doesnt like KE. not sure. all of the errors are simply deprecation notices, which shouldnt disrupt compiling. i think this was answered earlier but i cant find it
